package.json

```diff
@@ -50,10 +52,12 @@
     "@semantic-release/release-notes-generator": "13.0.0",
     "@types/node": "18.11.18",
     "@types/node-fetch": "2.6.11",
+    "@vitest/coverage-v8": "^1.6.0",
```

Comment:
if coverage is not used in the pipeline. I think the dependency is unnecesary yet